mailcheck/enabled.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  name: 'mailcheck is enable or disabled',
+  hypothesis: 'mailcheck will lead to higher confirmation rate of accounts',
+  startDate: '2015-01-01',
+  endDate: '2015-11-01',
+  subjectAttributes: ['uuid', 'isMetricsEnabled'],
+  independentVariables: ['mailcheckEnabled'],
+  eligibilityFunction: function (subject) {
+    return !! (subject && subject.isMetricsEnabled);
+  },
+  groupingFunction: function (subject) {
+    return {
+      mailcheckEnabled: this.bernoulliTrial(0.5, subject.uuid)
+    };
+  },
+  conclusion: {
+    mailcheckEnabled: false
```

Comment:
Only after the `endDate`. It isn't necessary to have a `conclusion` block from the get-go though.